sdk/openai/Azure.AI.OpenAI/README.md

```diff
@@ -713,7 +713,7 @@ File.WriteAllBytes("myAudioFile.mp3", response.Value.ToArray());
 
 The `gpt-4-vision-preview` model allows you to use images as input components into chat completions.
 
-To do this, provide distinct content items on the user message(s) for the chat completions request:
+To do this with an Url to the image, provide distinct content items on the user message(s) for the chat completions request:
```

Comment:
this is plain Englis, not C#, so don't use pascal casing for acronyms, i.e. URL, not Url.